dt-assets/js/shared-functions.js

```diff
@@ -129,8 +129,7 @@ window.TYPEAHEADS = {
     typeaheadSource : function (field, url) {
         return {
             contacts: {
-                display: [ "name", "ID"],
-                template: "<span>{{name}}</span>",
+                template: "<span>{{label}}</span>",
```

Comment:
I think this is the change that leads to this for the groups typeahead on the contacts:
![image](https://user-images.githubusercontent.com/24901539/71036892-dbe25480-211e-11ea-81ea-d05f70a2fce8.png)

Reply:
I can't seem to find a way to do conditional formatting for the template in typeahead. what I want to do is something like `if(label) { "<span>{{label}}</span>" } else { "<span>{{name}}</span>"` but that doesn't seem to work. 

The other option is to just add a label key to the groups endpoint like I did for the people groups. The conditional would live there then and if there is no translation label just use the post_title.

Reply:
We can create a "peopleGroupsTypeaheadSource" and use it for all the people group typeaheads.

Reply:
alternatively this might work:
```
`<span>{{ ${ label ? 'label' : 'name' } }}</span>`
```
But i prefer separating out the functionally since it is different enough

Reply:
should we make the TypeaheadSource more generic since we will probably use it for things like Locations, and Sources in the future? maybe something like "translatableTypeaheadSource"?